### PR TITLE
Add notice that WebDAV interface is not intended for browsers

### LINKED ISSUE #16359: OC 8.1: Add note when accessing remote.php/webdav
In OC 8.1 the SabreDAV browser on remote.php was removed. As this will causing confusion when users are not reading the release notes (which is often the case) please add a short note on this page like discussed in https://github.com/owncloud/core/issues/15648

### PR DESCRIPTION
**Pull request by LukasReschke**
Fixes #16359 as discussed in https://github.com/owncloud/core/issues/15648

Requires proper testing with sync clients. 

cc @jnfrmarks @rperezb Can you ensure some testing on this? THX.
cc @karlitschek As discussed. Please review.
cc @RealRancor Care to test as well? :smile:

### CODE REVIEW

File: lib/private/connector/sabre/dummygetresponseplugin.php [hunk 1]
```diff
@@ -42,15 +44,25 @@ class DummyGetResponsePlugin extends \Sabre\DAV\ServerPlugin {
...
 
 	/**
+	 * @param RequestInterface $request
+	 * @param ResponseInterface $response
 	 * @return false
 	 */
-	function httpGet() {
+	function httpGet(RequestInterface $request, ResponseInterface $response) {
+		$string = 'This is the WebDAV interface. It can only be accessed by ' .
+			'WebDAV clients such as the ownCloud desktop sync client.';
```
Comment (LukasReschke) [inline lib/private/connector/sabre/dummygetresponseplugin.php]: Native wording help requested :see_no_evil:

File: lib/private/connector/sabre/dummygetresponseplugin.php [hunk 2]
```diff
@@ -42,15 +44,25 @@ class DummyGetResponsePlugin extends \Sabre\DAV\ServerPlugin {
...
+		$stream = fopen('php://memory','r+');
+		fwrite($stream, $string);
+		rewind($stream);
+
+		$response->setBody($stream);
```
Comment (LukasReschke) [inline lib/private/connector/sabre/dummygetresponseplugin.php]: Hacky… alternative ideas? cc @DeepDiver1975

**Comment by PVince81** [comment]
I was wondering why it was still possible to download files with GET even with this plugin. The plugin doesn't check if the path is "/", might be safer to do so ?

Anyway, it still works to download files with GET, probably because to main server is run by default in such cases.

**Comment by PVince81** [comment]
I quickly debugged into the code and saw that Sabre's default handler for GET got called and returns false, which prevents running any further plugins.

:+1:

**Comment by LukasReschke** [comment]
> I quickly debugged into the code and saw that Sabre's default handler for GET got called and returns false, which prevents running any further plugins.

Yup. That's the black magic :crystal_ball:

**Comment by fcturner** [comment]
Is is possible to let this string be translated when using ownCloud in another language? 
On top of that; when using own branding, people might want to omit the 'such as the ownCloud desktop sync client'...

File: lib/private/connector/sabre/dummygetresponseplugin.php (hunk 1 above)
Comment (Xenopathic) [inline lib/private/connector/sabre/dummygetresponseplugin.php]: LGTM

**Comment by karlitschek** [comment]
nice :+1:

**Comment by MorrisJobke** [comment]
Works :+1:

**Pull request merged**